### DISCUSSION TITLE
how do i flx arm64?

### DISCUSSION THREAD

**Discussion by bendyfan12345** [KadeArchive/Kade-Engine#1712] [General]
it says ApplicationMain.exe and ARM64 how to i flx this?

**Comment by napstaa967** [comment]
there are 4 main processor types:
- x86
- amd64 (even if you have an intel processor its still amd64)
- arm
- arm64
make sure you typed:
`lime test windows`

if it still builds for arm64 it means either its configured wrong or you use arm64 and not amd64

if that's the case:
`lime test windows -64`
this will force lime to build to amd64 and not arm64

**Reply by bendyfan12345** [reply]
do you have Discord?

**Reply by NotIlulus** [reply]
didnt work, weird....

**Reply by NotIlulus** [reply]
can i config it back an alternate way without -64?
